### ISSUE TITLE
Drupal 7 always added jQuery and jQuery.once to the page

### ISSUE THREAD

**Issue by mglaman** [mglaman/retrofit-drupal#22]

**Comment by damienmckenna** [comment]
An initial task could be to just add documentation about this for when people are adding custom JS files to a new library.

**Comment by mglaman** [comment]
That is true. One idea I had was to have https://github.com/retrofit-drupal/retrofit/blob/main/src/Render/AttachmentResponseSubscriber.php always add the libraries (doesn't fix the fact jQuery.once is removed)

If folks wanted to disable this feature, there can be a service parameter to stop it from being added. So `sites/default/services.yml` could control how Retrofit works in this regard
